### DISCUSSION TITLE
How to change by less than 64??

### DISCUSSION THREAD

**Discussion by poisenbery** [comfyanonymous/ComfyUI#320] [Q&A]
I'm trying to make a 512x910 in comfy.

In WebUI, I can just type the number and it generates, but Comfy refuses to do steps smaller than 64.

How can I change numbers in increments of 1?

EDIT: I changed the nodes.py to allow latent h/w to have steps of 1, but now it errors when I generate lol.

So...I'm thinking there is some stuff that happens under the hood in Automatic1111 webui that allows non 64 multiples to be used as dimensions. What knowledge am I missing here?

**Comment by comfyanonymous** [comment]
This should be implemented now.
